src/main/java/main/FruitStoreMain.java

```diff
@@ -0,0 +1,19 @@
+package main;
+
+import core.basesyntax.db.CommandDao;
+import core.basesyntax.db.CommandDaoImpl;
+import model.FruitStorage;
+import model.FruitTransactionStorage;
+import service.FruitStoreService;
+
+public class FruitStoreMain {
+    public static void main(String[] args) {
+        CommandDao commandDao = new CommandDaoImpl();
+        FruitTransactionStorage fruitTransactionStorage = commandDao.read();
```

Comment:
This should just be List<FruitTransaction> . It's not a storage, naming is confusing. You don't need to store, Just parse input in to list, and then pass it to next service